content/docs/1.10.1/important-notes/_index.md

```diff
@@ -38,6 +39,76 @@ For the full release note, see [here](https://github.com/longhorn/longhorn/relea
     - [V2 Data Engine Replica Rebuild QoS](#v2-data-engine-replica-rebuild-qos)
     - [V2 Data Engine Volume Expansion](#v2-data-engine-volume-expansion)
 
+## Warning
+
```

Comment:
In general, the flow can be

- Mention what the upgrade from v1.8.x to v1.9.x does. 
- However, the v1beta1 leftovers might exist because some reasons....
- How to check and the steps before upgrading to v1.10.x